js/views/PageNav.js

```diff
@@ -140,7 +141,8 @@ export default class extends View {
   }
 
   navSettingsClick() {
-    // activate settings modal here
+    new SettingsModal().render().open();
```

Comment:
It's 3 lines of code to ensure duplicate settings models aren't launched. Why not put it in with this merge?